compiler/src/main/scala/org/scalajs/core/compiler/PrepJSExports.scala

```diff
@@ -466,6 +466,17 @@ trait PrepJSExports[G <: Global with Singleton] { this: PrepJSInterop[G] =>
           ignoreInvalid = false)
     }
 
+    allExportInfos.filter(_.destination == ExportDestination.Normal)
+      .groupBy(_.jsName).
+      filter { case (jsName, group) =>
```

Comment:
The dot on the previous line is probably not style conform :) I'll leave it to @sjrd though.

Reply:
Indeed, the dot should be at the beginning of this line.

Reply:
Fixed in https://github.com/scala-js/scala-js/pull/3633/commits/723f52104e4ab045cfe5719bc136bed7152353dc